src/types/correctSort.ts

```diff
@@ -0,0 +1,4 @@
+export interface IListOfGoodsHooks {
+  goods: string[];
+  classAlphabet: string;
+}
```

Comment:
The `IListOfGoodsHooks` interface is defined but not used in the `App` component as props. If this interface is intended to be used, ensure that the `App` component's props align with this interface. Otherwise, consider revising the interface or its intended usage.